### PR TITLE
Exercise 4

### PR DESCRIPTION
**Pull request by KrisGwynne**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: todo_app/data/trello_items.py
Comment (KrisGwynne) [inline todo_app/data/trello_items.py:12]: I had some real struggles trying to install python 3.10 correctly. It appeared to install correctly but when running poetry I would get errors missing the ssl module. May come back to this if I have time later down the line.

File: README.md
Comment (JackMeadDev) [inline README.md:68]: Nice clear instructions 👍

File: todo_app/data/trello_items.py
Comment (JackMeadDev) [reply todo_app/data/trello_items.py:12]: Sounds like sensible prioritisation to me; if you want to share at some point how you set up Python3.10 so we can take a look and see if we can advise then feel free to!

File: ansible-playbook.yml
Comment (JackMeadDev) [inline ansible-playbook.yml:33]: This should work but is quite permissive, it would be nice to try restricting this a bit more, for example setting the ec2-user as the owner and restricting write/execute permissions to other users;
```
      owner: ec2-user
      mode: '744'
```
I haven't tested this against your future commands so don't guarantee it works exactly as is though!

File: ansible-playbook.yml
Comment (JackMeadDev) [inline ansible-playbook.yml]: Usually it'd be nice to move this back to `main` before merging, but admittedly here we won't worry about keeping our Ansible up to date with future changes so feel free to leave this as is

**Review by JackMeadDev — APPROVED** [review]
Nice work, I can see your app is successfully deployed and your Ansible commands look clean & simple. Appreciate the comment about your step down from Python3.10 as well - makes it easy to understand the context there! I've left a few minor optional suggestions, but I'm happy for you to merge this when you're ready, and I'll tear down the VMs in the next few days so make sure you've got copies of anything you want to keep

**Pull request merged**
